### PR TITLE
Cmd-K shortcut mapped to Ctrl-L

### LINKED ISSUE #1041: want: ctrl-L and cmd-K support
With most Unix terminals, `ctrl-L` will clear/redraw the screen, and on macOS with either Terminal.app or iTerm2, `cmd-K` will clear the screen and flush the scroll back buffer. I do both of those quite a lot and I'd love to be able to do that with blink as well.

### PR DESCRIPTION
**Pull request by javierdemartin**

### CODE REVIEW

File: Blink/SpaceController.swift [hunk 1]
```diff
@@ -213,6 +213,13 @@ class SpaceController: UIViewController {
     _removeCurrentSpace()
   }
   
+  /**
+    Sends `Ctrl + L` sequence when `Cmd + K` is pressed. Mimics the iTerm's clean screen behavior. Behaves the same as `Ctrl + L`.
+   */
+  @objc private func _clearCurrentTerminal() {
+    currentDevice?.view.webView.reportPress(.control, keyId: "76:0")
```
Comment (yury) [inline Blink/SpaceController.swift:220]: I think we should add `l` to KeyCode enum. So instead of magic `76:0` it will be KeyCode.l.keyId

Comment (yury) [reply Blink/SpaceController.swift:220]: Actually we don’t need that code at all.

You need just add `0C` Hex here https://github.com/blinksh/blink/blob/449882bf526cd1aeefc1f8153e0ae22b766553dc/KB/Native/Model/KeyBindingAction.swift#L167

**Comment by say4n** [comment]
Hi @yury, any eta as to when this will be merged/available?

**Comment by yury** [comment]
@say4n I think, October

**Review by Dalvin12d — APPROVED** [review]

**Review by fizietechD — APPROVED** [review]

**Comment by carloscabanero** [comment]
Closing as it is outdated.

**Pull request closed without merging**